infra/build/functions/build_project.py

```diff
@@ -195,12 +195,18 @@ def get_env(fuzzing_language, build):
       'HOME': '/root',
       'OUT': build.out,
   }
+
   return list(sorted([f'{key}={value}' for key, value in env_dict.items()]))
 
 
 def get_compile_step(project, build, env, parallel):
   """Returns the GCB step for compiling |projects| fuzzers using |env|. The type
   of build is specified by |build|."""
+  set_git_repo_env = ''  #do nothing
```

Comment:
iirc what's going on here, it's hacky and we should do something else.
What I think you are doing is that when sanitizer is not "instrumentor" an empty string is inserted into the command on line 227.
But when it is "instrumentor" the command `&& export GITHUB_REPO...` is inserted.
IMO just change the entire command an each scenario, don't insert an empty string or `&& ...` that's really hard to understand.